packages/babel-parser/test/helpers/to-fuzzed-options.js

```diff
@@ -66,8 +73,6 @@ export default function toFuzzedOptions(options) {
     [randomIndex, 1, 0],
     [randomIndex, randomLine, randomColumn],
     [false, randomLine, false],
-    [false, false, randomColumn],
-    [randomIndex, false, false],
```

Comment:
Can we add `[randomIndex, false, randomIndex]` here? It seems that this option does not overlap with others.

Reply:
Great idea!